### DISCUSSION TITLE
RBAC on the free tier?

### DISCUSSION THREAD

**Discussion by notflip** [logto-io/logto#5636] [Q&A]
I'm building a product which might not ever be launched, so there is no budget for an auth tool right now.
I do however need roles and permissions, **is it correct that only 1 role is allowed in the free tier?**

**Comment by xiaoyijun** [comment]
Yes @notflip , for development purpose, you can use a dev tenant, which has unlimited roles and permissions.

FYI:
- [Tenant type](https://docs.logto.io/docs/recipes/tenant-type/#development)
- Create a dev tenant:
<img width="505" alt="image" src="https://github.com/logto-io/logto/assets/10806653/764dab36-c1fc-4d98-9da1-48db5a6cb8a4">
<img width="798" alt="image" src="https://github.com/logto-io/logto/assets/10806653/3ad850d6-114a-44e6-aa4e-597c0979a997">

**Reply by notflip** [reply]
Thanks! What are the limitations of the dev tenant? Can it be tested with multiple users/roles? Can it be tested on a live URL as well or only local?

**Reply by xiaoyijun** [reply]
The dev tenant will display a banner on the sign-in page, and will have MAU (maybe 100) in the future. Other quotas is totally the same as the pro plan in the production tenant and it can tested on a live URL.
